.circleci/config.yml

```diff
@@ -126,6 +126,12 @@ jobs:
         paths:
         - react-native-aztec/ios/Carthage
         - ~/.rncache
+    - run: 
```

Comment:
Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

Reply:
Hey @marecar3 seems you duplicated the `Build (if needed)` step here, and the bundle step here, removing your addition should resolve that

Reply:
Thanks @JavonDavis , removed!